prometheus-exporters-formula/metadata/form.yml

```diff
@@ -11,57 +11,61 @@ proxy_port:
   $disabled: '!formValues.proxy_enabled'
   $required: True
 
-node_exporter:
+exporters:
```

Comment:
If we ever wanted to store data about another type of "exporters" via this might not be unique enough. We could consider changing it to `prometheus_exporters` just to be more specific.

Reply:
Now after reviewing the screenshot this would probably not look very nice as we already have "Prometheus Exporters" as the formula name.

Reply:
I think it's fine to change the entry name and provide additional _name_ attribute for UI.